transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoopGroup.java

```diff
@@ -34,7 +34,7 @@
  * it only works on linux.
  */
 public final class EpollEventLoopGroup extends MultithreadEventLoopGroup {
-    {
+    static {
```

Comment:
I think there was reason for not using static... @carl-mastrangelo do you remember ?

Reply:
Delaying initialization for out-of-band static init.  If System.loadLibrary is called after EpollEventLoopGroup is referenced, there's no way for the Native methods to be linked correctly.

Reply:
Thank you, I will learn this point.

Reply:
Please revert